.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,15 +15,3 @@ https://github.com/kudobuilder/kudo/blob/master/RELEASE.md
 Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
 -->
 Fixes #
```

Comment:
- `Which issue(s) this PR fixes` and `Fixes #` are the same and we should keep one.
- `Does this PR introduce a user-facing change?` - I'd keep this one too. It would be easy to copy-paste it for the release notes.

Reply:
I can pull you some stats but I did not see people using that release notes column so far or it was something you would not put to release notes directly. We need to have that automation or I feel like it's waste of time even for me to fill that part and if it's waste of time, I don't need to have it in the PR template :slightly_smiling_face:
nobody is really manually going through the PRs when the release happens to copy this out because we have automatic changelog done by the go releaser from commit messages
which of course is not perfect :slightly_smiling_face: but I am not sure THAT is the part we need to improve the most right now :slightly_smiling_face:

and re: fixes - that need to be there because that is a keyword to github. The other is a heading - the PR is splitted into several sections, this one is for related tickets. I can drop the heading if we don't want to have the separation. That part I don't care about that much :slightly_smiling_face:

Reply:
Ok, you got me convinced there. I'd still remove `Which issue(s) this PR fixes:` since I don't know what I would put there (issues numbers are already in `Fixes #`).

Reply:
We're also reversing the relationship to have longer-lasting tracking issues and shorter lasting PRs.